runtime/mainnet/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -0,0 +1,335 @@
+
+//! Autogenerated weights for `pallet_xcm_benchmarks::generic`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 46.0.0
+//! DATE: 2025-02-24, STEPS: `2`, REPEAT: `1`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `tux`, CPU: `12th Gen Intel(R) Core(TM) i7-12700H`
+//! WASM-EXECUTION: `Compiled`, CHAIN: `None`, DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/pop-node
+// benchmark
+// pallet
+// --runtime
+// ./target/release/wbuild/pop-runtime-mainnet/pop_runtime_mainnet.compact.wasm
+// --pallet=pallet_xcm_benchmarks::generic
+// --extrinsic=report_holding,buy_execution,pay_fees,asset_claimer,query_response,transact,refund_surplus,set_error_handler,set_appendix,clear_error,descend_origin,execute_with_origin,clear_origin,report_error,claim_asset,trap,subscribe_version,unsubscribe_version,burn_asset,expect_asset,expect_origin,expect_error,expect_transact_status,query_pallet,expect_pallet,report_transact_status,clear_transact_status,set_topic,clear_topic,set_fees_mode,unpaid_execution
+// --repeat=1
+// --steps=2
+// --output=./runtime/mainnet/src/weights/xcm
+// --genesis-builder-preset=pop
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::Weight};
+use core::marker::PhantomData;
+
+/// Weight functions for `pallet_xcm_benchmarks::generic`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo<T> {
+	/// Storage: `ParachainInfo::ParachainId` (r:1 w:0)
+	/// Proof: `ParachainInfo::ParachainId` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
+	/// Storage: `ParachainSystem::HostConfiguration` (r:1 w:0)
+	/// Proof: `ParachainSystem::HostConfiguration` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainSystem::PendingUpwardMessages` (r:1 w:1)
+	/// Proof: `ParachainSystem::PendingUpwardMessages` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	pub(crate) fn report_holding() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `111`
+		//  Estimated: `1596`
+		// Minimum execution time: 291_401_000 picoseconds.
+		Weight::from_parts(291_401_000, 0)
```

Comment:
A little expensive compared to westend: https://github.com/paritytech/polkadot-sdk/blob/3dc3a11cd68762c2e5feb0beba0b61f448c4fc92/polkadot/runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_generic.rs#L72

Likely just hardware, but double checking regardless.

Reply:
Should be, I've did these benchmarks in my laptop :)
Recreating weights in our collators will give us a better point to compare.